k8s/k8s.go

```diff
@@ -69,8 +69,9 @@ func (rkd *runningK8sData) DaemonSets() ([]*models.K8sResource, error) {
 	}
 	ret := make([]*models.K8sResource, len(ds))
 	for i, d := range ds {
+		d2 := d // grab a value copy of "d" to enforce block scope heap reference, and avoid shadowing non-block scope "d"
```

Comment:
since `d` is a `*models.K8sResource`, this is gonna be a shallow copy. does this code work if you don't deep-copy?